src/config/constants.ts

```diff
@@ -34,7 +34,7 @@ export const ENV = {
 
 export const TREE_VIEW_MAX_WIDTH = 400;
 
-export const HOMEPAGE_NB_ELEMENTS_TO_SHOW = 12;
+export const HOMEPAGE_NB_ELEMENTS_TO_SHOW = 8;
```

Comment:
The number of cards on the page has changed ? is it not a multiple of 2, 3 and 4 ?

Reply:
The column is 3 max, so I can set 6 I guess?